### ISSUE TITLE
Table: Respect `workbench.reduceMotion`

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#169120]
From https://github.com/microsoft/vscode/issues/168464#issue-1484787768

There should be no hover animation whenever `workbench.reduceMotion` is true.
